about.html

```diff
@@ -0,0 +1,180 @@
+            that will leave you inspired, moved, and craving for more.<br />
+            Prepare to be captivated as Joshua Redman, the legendary jazz
+            saxophonist and composer, takes center stage, bringing his
+            unparalleled talent and passion to each performance.<br />
+            Don't miss this extraordinary opportunity to embark on a musical
+            journey like no other with the Joshua Redman World Tour 2023.
+          </p>
+        </div>
+        <p class="fs-18 dark ta-center fw-bold max-w-84">
+          Please contact us per Email for any further questions about Joshua
+          Redman WT 2023!<br />
+          <a href="#"><span class="fw-bolder underline-hover">contact@joshuaredmanwt2023.com
+              <i class="bi bi-chevron-right fs-16"></i></span></a>
+        </p>
+      </div>
+    </section>
+    <section class="grid bg-white border-light" id="embark">
+      <div class="container f-col g-30 ai-center js-center m-section max-w-75">
+        <h2 class="dark fs-30">
+          Embark with us<span class="orange">!</span><br />
+          <hr class="line-orange" />
+        </h2>
+        <p class="fs-18 max-w-84 ta-center">
+          <span class="fw-bold fs-italic">If you believe that our world tour should include your
+            city,</span>
+          join us on an incredible musical journey! we invite you to reach out
+          and let us know.
+          <a href="#"><span class="fs-18 fw-bold underline-hover">Contact us </span></a>
+          today to explore the possibilities of bringing our captivating
+          performances to your doorstep!
+        </p>
+        <p class="fs-18 max-w-84 ta-center fs-italic">
+          Thank you very much, <br />from Joshua Redman WT Team!
+        </p>
+        <img id="embark__logo" src="./visuals/logo-jrwt.png" />
+      </div>
+    </section>
+    <section class="grid" id="past-wt">
+      <div class="container dark f-col ai-center m-section g-30">
+        <h2 class="fs-30 ta-center">
+          See the past Joshua Redman World Tours<br />
+          <hr class="line-orange" />
+        </h2>
+        <p class="fs-18 ta-center">
+          Take a look to the last two World Tours of Joshua Redman.
+        </p>
+        <div class="f-row white w-75 g-10 jc-center" id="past-wt__container">
+          <a href="#">
+            <div class="f-col ta-center older-wt w-100 p-30 jc-center" id="wt-japan">
+              <h3 class="fs-40 ff-2 fw-lighter">2019</h3>
+              <span class="fs-18">Joshua Redman WT culmination in Japan</span>
+            </div>
+          </a>
+          <a href="#">
+            <div class="f-col ta-center older-wt w-100 p-30 jc-center" id="wt-brazil">
+              <h3 class="fs-40 ff-2 fw-lighter">2021</h3>
+              <span class="fs-18">Joshua Redman WT culmination in Brazil</span>
+            </div>
+          </a>
+        </div>
+      </div>
+    </section>
+    <footer class="grid bg-grey white fs-1">
```

Comment:
Great design @grauJavier 🤝

- [ ] As stated in the project requirements, kindly ensure your design is a replica of the design provided.
- Your design
![Your footer mobile about](https://github.com/grauJavier/capstone-project/assets/63917725/68ae9a06-e887-4d9e-b1f7-d515f9dd8b77)
- Given design
![Given-footer-mobile-about](https://github.com/grauJavier/capstone-project/assets/63917725/2f626c3b-f8d3-4bdd-9ead-0c075ef5288c)